# height: 1080
# width: 1440

CameraMatrix:
  fx: 1.2123364039628893e+03
  fy: 1.2067281918401040e+03
  cx: 7.3356989895379320e+02
  cy: 6.1888290521450563e+02

DistCoeffs:
  k1: -1.6696258062638117e-01
  k2: 8.1860005881057762e-01
  p1: -3.4991125703443364e-03
  p2: -2.3944263488253829e-03
  k3: -1.9937788347228935e+00

ReprojectionError: 5.2895421808764653e-01
